cli/src/main/java/org/openjdk/skara/cli/HgOpenJDKImport.java

```diff
@@ -36,16 +36,11 @@
 import java.util.logging.*;
 
 public class HgOpenJDKImport {
-    private static void die(Exception e) {
-        System.err.println(e.getMessage());
-        System.exit(1);
-    }
-
     private static Supplier<NoSuchElementException> error(String fmt, Object... args) {
```

Comment:
Pre-existing, but it's a bit unclear why NoSuchElementException is used - plain RuntimeException (or Exception) would probably make more sense? Or perhaps something like NoSuchFileException?

Reply:
Yes, that is a bit unclear to me as well, and I even wrote the code in the first place :laughing: I _think_ I wanted my `catch` clause to catch `NoSuchElementException` so I could just do `.orElseThrow()` or `.get()` and still display a somewhat nice error message automatically. Pretty bad idea in retrospect, I will rework this.